base/applications/taskmgr/procpage.c

```diff
@@ -931,3 +936,300 @@ int CALLBACK ProcessPageCompareFunc(LPARAM lParam1, LPARAM lParam2, LPARAM lPara
     }
     return ret;
 }
+
+static BOOL DevicePathToDosPath(LPWSTR lpPath, DWORD dwLength)
```

Comment:
It may be of interest to have the dwLength in number of bytes instead (for ease of usage of this function into the other one).

Reply:
OK.

Reply:
OK.

Reply:
OK.